proxy/inetaf_tcpproxy.go

```diff
@@ -0,0 +1,104 @@
+package proxy
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+
+	"github.com/inetaf/tcpproxy"
+)
+
+type childProxy struct {
+	l net.Listener
+	s Server
+}
+
+type InetAfTCPProxyServer struct {
+	Proxy    *tcpproxy.Proxy
+	children []*childProxy
+}
+
+// Close - implements Server - is this even called?
+func (tps *InetAfTCPProxyServer) Close() error {
+	_ = tps.Proxy.Close()
+	firstErr := tps.Proxy.Wait()
+	errChan := make(chan error)
```

Comment:
Shouldn't this be buffered, as it is in the `.Serve()` method below?  also probably a good idea to add a `defer close(errChan)`

Reply:
the defer close would not be necessary, but I agree it should be consistent with the other buffered error channels elsewhere.